xcm/v4/transfers.json

```
@@ -248,6 +248,19 @@
         },
         "instructions": "xtokensReserve"
       }
+    },
+    "ASTR": {
+      "chainId": "9eb76c5184c4ab8679d2d5d819fdf90b9c001403e9e17da2e14b6d8aec4029c6",
+      "multiLocation": {
+        "parachainId": 2006
+      },
+      "reserveFee": {
+        "mode": {
+          "type": "proportional",
+          "value": "20306692567"
```

that should be changed too, as that fee part connected to Astar network when it neither destination or source. It is not a case for our current scenario, but it would better to keep it right.
```suggestion
          "value": "1010366707520506354"
```

---

done, also in transfers_dev